### PR TITLE
Add HW for Class 1.

### PR DESCRIPTION
**Pull request by galaxyark**

### CODE REVIEW

File: FangzhouLu/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
+class1/.DS_Store
```
Comment (techbow) [inline FangzhouLu/.gitignore:2]: are you sure you want to check in the ignore file for git

File: FangzhouLu/class1/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!doctype html>
```
Comment (techbow) [inline FangzhouLu/class1/index.html]: by convention, doctype should be uppercase

File: FangzhouLu/class1/index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        </p>
+        <p>
+          Quisque accumsan eleifend erat et tristique. Vivamus convallis varius lobortis. Cras at ex at ex vulputate hendrerit non et lorem. Morbi a risus vitae tellus egestas posuere. Sed feugiat arcu ante, ac consequat turpis porta vitae. Integer ipsum orci, dignissim vel aliquet eget, faucibus eget ligula. Praesent turpis purus, vestibulum ac tellus non, mollis pulvinar libero.
+        </p>
+        <p>
+          Nullam nisl ligula, tempor eget augue pharetra, rhoncus accumsan nulla. Quisque non interdum nulla. Praesent at urna aliquam, sodales velit et, euismod felis. Quisque in urna justo. Nulla nec mollis sapien. Duis vel purus in turpis viverra egestas. Praesent dapibus faucibus leo, quis pretium enim finibus non. Cras porttitor lacinia sapien, id faucibus mi porttitor vel. Aliquam erat volutpat. Sed id porta eros.
+        </p>
+      </div>
+    </artical>
```
Comment (techbow) [inline FangzhouLu/class1/index.html]: you define this tag? or just typo article

Comment (galaxyark) [inline FangzhouLu/class1/index.html]: typo

File: FangzhouLu/.gitignore (hunk 1 above)
Comment (galaxyark) [inline FangzhouLu/.gitignore:2]: I am.

**Pull request merged**